css/decanter-kss.css

```diff
@@ -0,0 +1,434 @@
+/*! normalize.css v8.0.0 | MIT License | github.com/necolas/normalize.css */
```

Comment:
What is this new file for?

Reply:
This is where we are adding styles for the style guide only. Is there a better place to be doing this?

Reply:
If this is specific to the display of something in the styleguide we should create our custom kss theme so that we keep this work separated and clear.

Reply:
This might be a nice theme to start from: https://www.npmjs.com/package/kss-orion-theme